src/map/battle.cpp

```diff
@@ -3406,7 +3408,11 @@ static void battle_calc_multi_attack(struct Damage* wd, struct block_list *src,s
 			if (sc && sc->data[SC_KAGEMUSYA])
 				max_rate = sc->data[SC_KAGEMUSYA]->val1 * 10; // Same rate as even levels of TF_DOUBLE
 			else
+#ifdef RENEWAL
+				max_rate = max(7 * skill_lv, sd->bonus.double_rate);
```

Comment:
qq, does this apply to all thief base class? or will it only affect the gx?

Reply:
```
[Lv 1] : 발동확률: 7%, 명중보정: 1%
[Lv 2] : 발동확률: 14%, 명중보정: 2%
[Lv 3] : 발동확률: 21%, 명중보정: 3%
[Lv 4] : 발동확률: 28%, 명중보정: 4%
[Lv 5] : 발동확률: 35%, 명중보정: 5%
[Lv 6] : 발동확률: 42%, 명중보정: 6%
[Lv 7] : 발동확률: 49%, 명중보정: 7%
[Lv 8] : 발동확률: 56%, 명중보정: 8%
[Lv 9] : 발동확률: 63%, 명중보정: 9%
[Lv10] : 발동확률: 70%, 명중보정: 10%
```